common/components/Root/index.jsx

```diff
@@ -1,17 +1,20 @@
 import React, { Component } from 'react';
```

Comment:
we need //@flow here, can you lurk around for eslint rule for that or something?